package.json

```diff
@@ -66,6 +66,7 @@
     "@types/redux": "3.6.0",
     "@types/redux-localstorage": "1.0.8",
     "@types/rimraf": "2.0.3",
+    "@types/valid-url": "1.0.2",
```

Comment:
Where does this get used?

Reply:
In [sanitize-html](https://github.com/Automattic/simplenote-electron/pull/1884/files#diff-70d173f4f222b85be1283cc7b497680cR2)